src/theme/themeContext.tsx

```diff
@@ -0,0 +1,3 @@
+import React from 'react';
+
+export const ThemeContext = React.createContext({});
```

Comment:
usually we define type for context createContext<...>({theme: Themes.light, setTheme: (theme) => {} });